tests/src/OneLogin/saml2_tests/authn_request_test.py

```diff
@@ -255,7 +255,7 @@ def testCreateEncSAMLRequest(self):
         inflated = decompress(decoded, -15)
 
         self.assertRegexpMatches(inflated, '^<samlp:AuthnRequest')
-        self.assertRegexpMatches(inflated, 'AssertionConsumerServiceURL="http://stuff.com/endpoints/endpoints/acs.php">')
+        self.assertRegexpMatches(inflated, 'AssertionConsumerServiceURL="http://stuff.com/endpoints/endpoints/acs.php"(\s)*>')
```

Comment:
The RegExp unit tests were breaking caus of the presence of a training whitespace in Circle-CI. The \s will ignore all whitespaces. (e.g. when attributeConsumingService is not there)

yeah...i have yet to add unit tests...